scripts/e2e_tests_partner_plugin_mock_logs.sh

```
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+# Set the target directory with tilde for home
+target_dir="~/tmp/cmb-e2e"
```

Shellcheck (0.10.0)

[warning] Tilde does not expand in quotes. Use $HOME.

(SC2088)